contracts/DXswapFeeReceiver.sol

```diff
@@ -0,0 +1,127 @@
+pragma solidity =0.5.16;
+
+import './interfaces/IDXswapFactory.sol';
+import './interfaces/IDXswapPair.sol';
+import './interfaces/IWETH.sol';
+import './libraries/TransferHelper.sol';
+import './libraries/SafeMath.sol';
+
+
+contract DXswapFeeReceiver {
+    using SafeMath for uint;
+
+    address public owner;
+    IDXswapFactory public factory;
+    address public WETH;
+    address public ethReceiver;
+    address public fallbackReceiver;
+
+    constructor(
+        address _owner, address _factory, address _WETH, address _ethReceiver, address _fallbackReceiver
+    ) public {
+        owner = _owner;
+        factory = IDXswapFactory(_factory);
+        WETH = _WETH;
+        ethReceiver = _ethReceiver;
+        fallbackReceiver = _fallbackReceiver;
+    }
+    
+    function() external payable {}
+
+    function transferOwnership(address newOwner) external {
+        require(msg.sender == owner, 'DXswapFeeReceiver: FORBIDDEN');
+        owner = newOwner;
+    }
+    
+    function changeReceivers(address _ethReceiver, address _fallbackReceiver) external {
+        require(msg.sender == owner, 'DXswapFeeReceiver: FORBIDDEN');
+        ethReceiver = _ethReceiver;
+        fallbackReceiver = _fallbackReceiver;
+    }
+    
+    // Returns sorted token addresses, used to handle return values from pairs sorted in this order
```

Comment:
I think we could make this comment more clear.

```used to handle return values from pairs sorted in this order``` I think is a bit confusing.  We should probably mention that this ordering is consistent with how DXswap pairs are created, here: https://github.com/levelkdev/dxswap-core/blob/master/contracts/DXswapFactory.sol#L27

Reply:
Just discovered the code lives here: https://github.com/levelkdev/dxswap-periphery/blob/master/contracts/libraries/DXswapLibrary.sol#L11